blocks/init/src/blocks/custom/column/components/column-responsive-tab-content.js

```diff
@@ -54,7 +55,8 @@ export const ColumnResponsiveTabContent = (props) => {
               {__('Offset', 'eightshift-boilerplate')}
             </Fragment>
           }
-          help={sprintf(__('Option to change offset block in grid from the left. Change block offset in %d columns range. Example 6 = 50% screen width. If you set a value to -1 it will not be used and the parent brakepoint will be used.', 'eightshift-boilerplate'), globalSettings.maxCols)}
+          help={sprintf(__('Option to change offset block in grid from the left. Change block offset in %d columns range. Example 6 = 50 percent of the screen width. If you set a value to -1 it will not be used and the parent brakepoint will be used.', 'eightshift-boilerplate'), globalSettings.globalVariables.maxCols)}
```

Comment:
```suggestion
          help={sprintf(__('Option to change the block offset in the grid from the left. Change block offset in %d columns range. Example: 6 is 50\% of the screen width. If you set a value to -1 it will not be used and the parent breakpoint will be used.', 'eightshift-boilerplate'), globalSettings.globalVariables.maxCols)}
```